Telegram_bot/telegram_bot.py

```diff
@@ -0,0 +1,30 @@
+from sympy import Symbol,collect
+from telegram import Update
+from telegram.ext import ApplicationBuilder, CommandHandler, ContextTypes
+
+async def sum(update: Update, context:ContextTypes.DEFAULT_TYPE)-> None:
+    user_text = update.message.text
+    user_text = user_text.replace("/add",'')
+    user_text = user_text.replace('^','**').replace("x",'*x').replace('-0', '')
```

Comment:
Чтобы заработало, нужно сделать так.
```suggestion
    user_text = user_text.replace("/sum",'')  # у вас ведь эта команда, а не /add
    user_text = user_text.replace('^','**').replace("x",'*x').replace('=0', '')
```